src/pages/developing/frameworks/web-components.mdx

```diff
@@ -160,9 +160,12 @@ View the `<dds-dotcom-shell>` in [Storybook Web Components](http://www.ibm.com/s
 </Column>
```

Comment:
You could remove line 74 `Available tags:` 
The paragraph above does a great job of introducing the table. :) 

Lines 82 and 95
Should it be `Here are examples of setting by tag:` 
Suggestion to replace `on` with `of` unless I'm not understanding something. 

Line 101
Suggestion to remove `what is` from the sentence. 
Check npmjs.com to see ~what is~ the latest available version. 

Line 138
I'd change the link to appear as follows, and add a period to the end. 
Learn more about <Link to="/developing/carbon-cdn-style-helpers">Carbon CDN style helpers</Link>.
Okay, that's not showing... I was trying to link only the words `Carbon CDN style helpers` in the sentence.